src/game.cpp

```diff
@@ -12067,21 +12066,25 @@ void game::quicksave()
 void game::quickload()
 {
     const WORLD *active_world = world_generator->active_world;
+    std::string const world_name = active_world->world_name;
```

Comment:
This will trigger UB, as apparently `active_world` can be null. Note the check for null is *after* it gets dereferenced here. You need to move this line after the if clause.

Reply:
Great catch! I can't believe I missed that with the explicit nullptr check.